dbt_coves/tasks/generate/base.py

```diff
@@ -24,13 +24,12 @@ class BaseGenerateTask(BaseConfiguredTask):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.metadata = None
+        self.prop_files_created_by_dbtcoves = list()
```

Comment:
Lists are usually initializated as `[]`. Anyway, the right structure I think is `set()` as we don't need duplicated items.